packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -558,6 +558,15 @@ export function props(
       } else if (styleProp === 'paddingInlineEnd') {
         flatStyle.paddingEnd = styleValue;
       }
+      // visibility polyfill
+      // note: we can't polyfill nested visibility changes
+      else if (styleProp === 'visibility') {
+        if (styleValue === 'hidden' || styleValue === 'collapse') {
+          flatStyle.opacity = 0;
```

Comment:
Do we have something like `aria-hidden` in RN?

Reply:
Yeah, we have a polyfill for `aria-hidden` in RN itself. I'll add that prop to the polyfill